### PR TITLE
Add workbooks support for new entry scope

### PR DESCRIPTION
**Pull request by artemipanchuk**

### CODE REVIEW

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/dash-main-view-1501/ru/">dash.main.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/dash-main-view-1501/en/">dash.main.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/new-workbooks-1501/ru/">new-workbooks (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/new-workbooks-1501/en/">new-workbooks (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/new-workbooks-empty-table-1501/ru/">new-workbooks.empty-table (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/new-workbooks-empty-table-1501/en/">new-workbooks.empty-table (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/dash-main-view-1501/ru/">dash.main.view (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/dash-main-view-1501/en/">dash.main.view (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/new-workbooks-1501/ru/">new-workbooks (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/new-workbooks-1501/en/">new-workbooks (en)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10280-1-1501/new-workbooks-empty-table-1501/en/">new-workbooks.empty-table (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

File: src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx [hunk 1]
```diff
@@ -154,7 +156,8 @@ export const WorkbookEntriesTable = React.memo<WorkbookEntriesTableProps>(
             });
         };
 
-        const [dashChunk = [], connChunk = [], datasetChunk = [], widgetChunk = []] = chunks;
+        const [dashChunk = [], connChunk = [], datasetChunk = [], widgetChunk = [], reportsChunk] =
```
Comment (imsitnikov) [inline src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx:159]: `reportsChunk` -> `reportsChunk = []`?

File: src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx [hunk 2]
```diff
@@ -264,6 +267,28 @@ export const WorkbookEntriesTable = React.memo<WorkbookEntriesTableProps>(
                             onShowRelatedClick={onShowRelated}
                         />
                     )}
+
+                    {showDataEntities && availableScopes?.includes(EntryScope.Presentation) && (
```
Comment (imsitnikov) [inline src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx:271]: Is the "Limited Viewer" role not enough to use reports? Reports are similar to dashboards and should have a similar access model, isn't it?

**Review by imsitnikov — CHANGES_REQUESTED** [review]

File: src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx [hunk 3]
```diff
@@ -264,6 +267,28 @@ export const WorkbookEntriesTable = React.memo<WorkbookEntriesTableProps>(
...
+                        <MainTabContent
+                            chunk={reportsChunk}
+                            actionCreateText={i18n('action_create-report')}
+                            title={i18n('title_reports')}
+                            actionType={CreateEntryActionType.Report}
+                            isShowMoreBtn={Boolean(
+                                reportsChunk?.length > 0 && mapTokens?.[EntryScope.Presentation],
```
Comment (Sergey-weber) [inline src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx:278]: EntryScope.Report?

File: src/shared/constants/navigation.ts [hunk 4]
```diff
@@ -6,6 +6,8 @@ export const PLACE = {
     WIDGETS: 'widgets',
     CONNECTIONS: 'connections',
     CLUSTERS: 'clusters',
+    REPORTS: 'reports',
+    // TODO delete after update
     PRESENTATIONS: 'presentations',
```
Comment (artemipanchuk) [inline src/shared/constants/navigation.ts:11]: Can be deleted after platform update

File: src/shared/constants/navigation.ts [hunk 5]
```diff
@@ -17,5 +19,7 @@ export const MAP_PLACE_TO_SCOPE = {
     [PLACE.WIDGETS]: 'widget',
     [PLACE.CONNECTIONS]: 'connection',
     [PLACE.CLUSTERS]: 'clusters',
-    [PLACE.PRESENTATIONS]: 'presentation',
+    [PLACE.REPORTS]: 'report',
+    // TODO delete after update
```
Comment (artemipanchuk) [inline src/shared/constants/navigation.ts:23]: Can be deleted after platform update

File: src/shared/types/common.ts [hunk 6]
```diff
@@ -251,6 +251,9 @@ export enum EntryScope {
     Dataset = 'dataset',
     Folder = 'folder',
     Connection = 'connection',
+    Report = 'report',
+
+    // TODO: delete after update
```
Comment (artemipanchuk) [inline src/shared/types/common.ts:256]: Can be deleted after platform update

File: src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx (hunk 3 above)
Comment (artemipanchuk) [reply src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx]: Fixed

File: src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx (hunk 2 above)
Comment (artemipanchuk) [reply src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx]: Fixed

File: src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx (hunk 1 above)
Comment (artemipanchuk) [reply src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx]: Fixed

File: src/ui/units/workbooks/components/CreateEntry/utils.tsx [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+import {EntryScope, Feature} from 'shared';
+import {EntityIcon} from 'ui/components/EntityIcon/EntityIcon';
+import Utils from 'ui/utils';
+
+import {CreateEntryActionType} from '../../constants';
+
+const i18n = I18n.keyset('new-workbooks');
+const b = block('dl-workbook-create-entry');
+
+export const useCreateEntryOptions = ({
```
Comment (Sergey-weber) [inline src/ui/units/workbooks/components/CreateEntry/utils.tsx:15]: is it hook should be here? useCreateEntryOptions in regestry

**Review by Sergey-weber — APPROVED** [review]

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**